Dockerfile

```diff
@@ -36,16 +36,17 @@ RUN cd /home/jenkins/bin && \
   curl -LJ https://github.com/aquasecurity/tfsec/releases/download/v0.56.0/tfsec-linux-amd64 -o tfsec && \
   chmod 755 tfsec
 
-RUN curl -L "$(curl -s https://api.github.com/repos/accurics/terrascan/releases/latest | grep -o -E "https://.+?_Linux_x86_64.tar.gz")" > terrascan.tar.gz && \
+RUN curl -L "$(curl -s https://api.github.com/repos/tenable/terrascan/releases/latest | grep -o -E "https://.+?_Darwin_x86_64.tar.gz")" > terrascan.tar.gz && \
   tar -xf terrascan.tar.gz terrascan && rm terrascan.tar.gz && \
   install terrascan /home/jenkins/bin && rm terrascan && \
   chmod 755 /home/jenkins/bin/terrascan
     
 ENV PATH="/home/jenkins/bin:${PATH}"
 
 # Install EB CLI
+RUN pip install --user virtualenv
+ENV PATH="/home/jenkins/.local/bin:${PATH}"
 RUN git clone https://github.com/aws/aws-elastic-beanstalk-cli-setup.git \
-  && ./aws-elastic-beanstalk-cli-setup/scripts/bundled_installer \
+  && python ./aws-elastic-beanstalk-cli-setup/scripts/ebcli_installer.py \
```

Comment:
Reference: https://github.com/aws/aws-elastic-beanstalk-cli-setup  and https://github.com/Vestmark/jnlp-microservices/blob/main/Dockerfile#L66-L70